adapter/src/main/java/ar/edu/undec/adapter/data/curso/crud/CrearCursoCrud.java

```diff
@@ -0,0 +1,13 @@
+package ar.edu.undec.adapter.data.curso.crud;
+
+import ar.edu.undec.adapter.data.curso.model.CursoEntidad;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface CrearCursoCrud extends CrudRepository<CursoEntidad, Integer> {
```

Comment:
Aqui el tipo del atributo ID no es Integer, sino UUID
```suggestion
public interface CrearCursoCrud extends CrudRepository<CursoEntidad, UUID> {
```